invenio_rdm_records/services/config.py

```diff
@@ -135,6 +139,28 @@ class RDMRecordCommunitiesConfig(ServiceConfig, ConfiguratorMixin):
     max_number_of_removals = 100
 
 
+class RDMCommunityRecordsConfig(BaseRecordServiceConfig, ConfiguratorMixin):
```

Comment:
Comment for reviewers: In this case we are inheriting from the recod service, mainly because we alos have implmented the search records inside a community. For this reason it makes sense to me that it's a record based service. However feel free to challenge this!

Reply:
could you write a comment in the code explaining this ?